benchmark/v4.0-performance-benchmarking-with-tpch.md

```diff
@@ -85,13 +85,17 @@ logger.level: "info"
 learner_config.log-level: "info"
 ```
 
-##### 全局变量配置
+##### 变量配置
+
+> **注意：**
+>
+> 部分参数为 SESSION VARIABLE，建议所有 QUERRY 都需要在当前 SESSION 中执行。
```

Comment:
```suggestion
> 部分参数为 SESSION 变量。建议所有查询都在当前 SESSION 中执行。
```

Reply:
the suggestion loos good to me, @glkappe please address this comment.